test/extensions/MockComponentPartial.test.js

```diff
@@ -0,0 +1,92 @@
+const brs = require("brs");
+const { ValueKind, Callable, BrsString, RoAssociativeArray } = brs.types;
+
+const { ComponentDefinition } = require("../../lib/componentprocessor");
+
+const { Interpreter } = require("../../lib/interpreter");
+const { mockComponentPartial } = require("../../lib/extensions/mockComponentPartial");
+const { BrsInvalid } = require("../../lib/brsTypes/BrsType");
+
+describe("mockComponentPartial", () => {
+    describe("methods", () => {
```

Comment:
Probably don't need this inner `describe` since there's only one thing inside it, but I'll remove it before I merge 😃